plugins/firebase/tsup.config.ts

```diff
@@ -0,0 +1,25 @@
+import { defineConfig } from "tsup";
+
+export default defineConfig([
+  {
+    entry: ["./src/index.ts"],
+    format: ["esm", "cjs"],
+    outDir: "dist",
+  },
+  {
+    entry: ["firebase/functions/index.ts"],
+    format: ["cjs"],
+    outDir: "dist/firebase",
+    noExternal: ["@hot-updater/core"],
```

Comment:
```suggestion
```

Since `@hot-updater/core` is listed in dependencies, it is automatically treated as noExternal, so it’s not necessary to specify it manually.